src/test/java/com/aws/greengrass/certificatemanager/certificate/CertificateExpiryMonitorTest.java

```diff
@@ -121,4 +124,42 @@ public void GIVEN_certs_added_to_monitor_WHEN_expired_THEN_regenerated() throws
         X509Certificate cert2fourth = cg2.getCertificate();
         assertThat(cert2fourth, is(cert2third));
     }
+
+    @Test
+    public void GIVEN_certs_with_max_duration_WHEN_time_greater_than_default_THEN_not_generated() throws Exception {
+        Mockito.when(mockConfigurationTopics.findOrDefault(any(), any())).thenReturn(CertificatesConfig.MAX_SERVER_CERT_EXPIRY_SECONDS);
```

Comment:
You're making assumptions about the values of these constants and the test could easily break if they change. Better to be explicit